### PR TITLE
Add message box for warning and js to hide/show it for #2469

### PR DESCRIPTION
**Pull request by mswindler**
https://github.com/Strategy11/formidable-pro/issues/2469

### CODE REVIEW

File: js/formidable_admin.js [hunk 1]
```diff
@@ -5552,6 +5571,8 @@ function frmAdminBuildJS() {
 			//BCC, CC, and Reply To button functionality
 			$formActions.on( 'click', '.frm_email_buttons', showEmailRow );
 			$formActions.on( 'click', '.frm_remove_field', hideEmailRow );
+			$formActions.on( 'change', '.frm_to_row', showEmailWarning );
+			$formActions.on( 'change', '.frm_from_row', showEmailWarning );
```
Comment (stephywells) [inline js/formidable_admin.js:5575]: These two lines can be combined into one.

File: classes/views/frm-form-actions/_email_settings.php [hunk 2]
```diff
@@ -40,6 +40,10 @@
 	<input type="text" name="<?php echo esc_attr( $this->get_field_name( 'from' ) ); ?>" value="<?php echo esc_attr( $form_action->post_content['from'] ); ?>" class="frm_not_email_to frm_email_blur large-text <?php FrmAppHelper::maybe_add_tooltip( 'from', 'open' ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'from' ) ); ?>" />
 </p>
 
+<p class="frm_email_row frm_from_to_match_row <?php echo ( ( $form_action->post_content['from'] !== $form_action->post_content['email_to'] ) ? 'frm_hidden' : '' ); ?>" data-emailrow="from_to_warning">
```
Comment (stephywells) [inline classes/views/frm-form-actions/_email_settings.php:43]: Let's add some styling to this. How about the frm_error_style class?

File: js/formidable_admin.js [hunk 3]
```diff
@@ -3457,6 +3457,25 @@ function frmAdminBuildJS() {
...
+	function showEmailWarning() {
+		/*jshint validthis:true */
+		var actionBox = jQuery( this ).closest( '.frm_form_action_settings' ),
+			emailRowSelector = '.frm_from_to_match_row',
+			match = false,
+			fromVal = actionBox.find( 'input[name$="[post_content][from]"]' ).val(),
+			toVal = actionBox.find( 'input[name$="[post_content][email_to]"]' ).val();
+
+		if ( fromVal === toVal ) {
+			match = true;
```
Comment (stephywells) [inline js/formidable_admin.js:3469]: Perhaps we can simplify this a bit by getting rid of the match var. We can instead show the message here.

File: classes/views/frm-form-actions/_email_settings.php (hunk 2 above)
Comment (mswindler) [reply classes/views/frm-form-actions/_email_settings.php]: Added and removed

**Comment by stephywells** [comment]
Thank you @mswindler ! Looks great!

**Pull request merged**